File: OOP/PrimeNumber.java
import java.util.Scanner;

public class PrimeNumber {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.println("Enter a number: ");
        int n = s.nextInt();

        int temp = 0;

        for(int i = 2; i < n; i++) {
            if(n % i == 0) {
                temp++;
            }
        }

        if(temp > 0) {
            System.out.println("NON-PRIME NUMBER!");
        } else {
            System.out.println("PRIME NUMBER!");
        }

        s.close();
    }
}

File: OOP/PrimeNumber.py
def prime_num(num):
    temp = 0
    for i in range(2, num):
        if num % i == 0:
            temp = temp + 1

    if temp > 0:
        print(f"{num} is NON-PRIME NUMBER!")
    else:
        print(f"{num} is PRIME NUMBER!")


num = int(input("Enter a number to check Prime Number: "))
prime_num(num)
